Doc/whatsnew/3.12.rst

```diff
@@ -982,6 +982,11 @@ Porting to Python 3.12
   effects, these side effects are no longer duplicated.
   (Contributed by Victor Stinner in :gh:`98724`.)
 
+* The interpreter's error indicator is now always normalized. This means
+  that :c:func:`PyErr_SetObject`, :c:func:`PyErr_SetString` and the other
+  functions that set the error indicator now need to normalize the exception
```

Comment:
nit: including "need to" gives a whiff of "calling code must do something different," which isn't really the point here: the point is those functions _will_ normalize the exception before storing it
```suggestion
  functions that set the error indicator now normalize the exception
```